src/transformStateWithClones.js

```diff
@@ -7,7 +7,37 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const cloneObject = {};
+  const arrayWithObjects = [];
```

Comment:
The name of the variable should tell us about the information that is stored inside, not its data type
So I would do it like that:
```suggestion
  const stateVersions = [];
```